package.json

```
@@ -109,9 +106,9 @@
       ]
     },
     "server-headless": {
-      "command": "./runBiDiServer.sh --headless=true",
+      "command": "./tools/run-bidi-server.mjs --headless=true",
       "files": [
-        "runBiDiServer.sh"
+        "tools/run-bidi-server.mjs"
```

Be consistent here or down.
```suggestion
        "./tools/run-bidi-server.mjs"
```